### PR TITLE
Added Support for Python 3.5 and Django 1.10

### LINKED ISSUE #14: Package is not Python 3.5-compatible
This package was added to edx/course-discovery (https://github.com/edx/course-discovery/pull/150); but, this package is not compatible with Python 3.5.

Make this package compatible with both Python 2.7 and 3.5.

### PR DESCRIPTION
**Pull request by clintonb**
Fixed #14

### CODE REVIEW

**Comment by clintonb** [comment]
@doctoryes @efischer19 @macdiesel please review and merge

File: setup.py [hunk 1]
```diff
@@ -1,53 +1,26 @@
...
-
+import release_util
 
-long_description = io.open('README.md', encoding='utf-8').read()
+long_description = io.open('README.rst', encoding='utf-8').read()
 
 METADATA = dict(
     name='edx-django-release-util',
-    version='0.0.5',
+    version=release_util.__version__,
```
Comment (efischer19) [inline setup.py]: Will moving the version from 0.0.5 to 0.0.1 confuse PyPI publishing?

https://pypi.python.org/pypi/edx-django-release-util/0.0.5

Comment (clintonb) [inline setup.py]: Jumping to 0.1.0. There should be no issues.

Comment (efischer19) [inline setup.py]: My mistake, misread that. Carry on!

**Comment by efischer19** [comment]
👍

**Comment by doctoryes** [comment]
@jmbowman This PR seems relevant to your PR here:

https://github.com/edx/cookiecutter-django-app/pull/1

which I wish I could have used to boot up this repo. Do the changes match up with your cookiecutter repo?

File: setup.py [hunk 2]
```diff
@@ -1,53 +1,26 @@
...
     long_description=long_description,
     license='AGPL',
     url='http://github.com/edx/edx-django-release-util',
-    install_requires=load_requirements('requirements/base.txt',),
-    tests_require=load_requirements('requirements/test.txt'),
+    install_requires=[
+        'django>=1.8,<1.11',
+        'PyYAML>=3.11',
+        'six>=1.10.0,<2.0.0',
+    ],
```
Comment (macdiesel) [inline setup.py]: Any way we can make this more compatible with the pip requirements files so we do not need to update the dependencies in 2 places?

Comment (clintonb) [inline setup.py]: Those files no longer exist. This is the only place to list package requirements.

File: setup.py [hunk 3]
```diff
@@ -59,7 +32,12 @@ def load_requirements(*requirements_paths):
         'License :: OSI Approved :: GNU Affero General Public License v3',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.5',
```
Comment (jmbowman) [inline setup.py]: "Programming Language :: Python :: 3" should also be included in order to appear on the Python 3 packages list on PyPI; more details at https://github.com/brettcannon/caniusepython3#how-do-you-tell-if-a-project-has-been-ported-to-python-3

Comment (clintonb) [inline setup.py]: Updated.

**Comment by macdiesel** [comment]
👍

**Comment by jmbowman** [comment]
👍

**Comment by clintonb** [comment]
Thanks all. @macdiesel I don't have push/merge rights on this repo. Would you merge this PR, and make a new release/tag for 0.1.0, please?

**Pull request merged**